bodhi/tests/client/test_bindings.py

```diff
@@ -191,9 +191,9 @@ class TestBodhiClient_override_str(unittest.TestCase):
     """
```

Comment:
Optional: It would be good to add two tests for the two new bindings API flag. We do have tests that hit that code through the CLI, but the bindings are their own interface that some third parties use too, so it would be good to test them independently. This is not required to merge.